superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -136,16 +138,8 @@ def get_recipe_text(self, distributor, license_text, die_msg=None):
         elif isinstance(self.license, list):
             self.license = self.license[0].replace(' ', '-')
```

Comment:
@shaneallcroft this hackyness needs to be killed with fire. This just removes the list entirely and replaces it with its first element.

Therefore, this line and the one that immediately follows should be removed.

Reply:
You got it! :+1: